apps/E2E/src/common/BasePage.ts

```diff
@@ -72,6 +83,59 @@ export abstract class BasePage {
   /**************** UI Element Interaction Methods ******************/
   /******************************************************************/
 
+  /**
+   * For any given page object, this method automates:
+   *  - Navigating to the page by clicking its component button
+   *  - Waiting for the component page to load
+   *  - Optionally expanding its E2E sections IF a boolean flag is passed.
+   *
+   * This also contains error checking through its waiters, removing the extra `expect()` calls during test setup.
+   *
+   * @param {boolean} showE2ESection (default = `false`) Some pages have E2E tests that only check if the component causes
```

Comment:
Nit:

Some components' E2E tests check only if the test page loads correctly or not. Others (majority), perform UI manipulation tests on UI components on the test page. In these scenarios, these UI components have their own section on the test page (by default, it's hidden so partners don't see it). If this parameter is true, this method opens up that testing section.